pkg/auth/model/model.go

```diff
@@ -41,23 +41,19 @@ type User struct {
 }
 
 // hashPassword generates a hashed password from a plaintext string
-func hashPassword(password string) ([]byte, error) {
-	pw, err := bcrypt.GenerateFromPassword([]byte(password), bcrypt.DefaultCost)
-	if err != nil {
-		return nil, err
-	}
-	return pw, nil
-}
-
 func (u *User) UpdatePassword(password string) error {
-	pw, err := hashPassword(password)
+	pw, err := HashPassword(password)
 	if err != nil {
 		return err
 	}
 	u.EncryptedPassword = pw
 	return nil
 }
 
+func HashPassword(password string) ([]byte, error) {
```

Comment:
the code doc left above and does not match the func case.

Reply:
Not code I wrote but moved the description to the proper function.